php-practice_1.php

```diff
@@ -0,0 +1,126 @@
+<?php
+// Q1 変数と文字列
+$name = '奥西';
+echo  '私の名前は「'. $name .'」です。' . "\n";
+
+// Q2 四則演算
+$num = 5 * 4;
+echo $num . "\n";
+$num /= 2;
+echo $num . "\n";
+
+// Q3 日付操作
+date_default_timezone_set('Asia/Tokyo');
+echo date('現在時刻は、Y年m月d日 H時i分s秒です。') . "\n";
+
+// Q4 条件分岐-1 if文
+$device = 'mac';
+if ($device === 'windows') {
+  echo '使用OSは、windowsです。' . "\n";
```

Comment:
[imo]
この書き方でも問題ないですが、下記にすることでif文の条件式を変更し
ネストを減らすことができます！
```suggestion
  echo '使用OSは、' . $device . 'です。' . "\n";
```

Reply:
ご指摘頂いた対応をし、if文のネストを減らしました。